cmake/BLTMacros.cmake

```diff
@@ -796,7 +796,8 @@ macro(blt_add_executable)
     list(GET arg_SOURCES 0 _first)
     get_source_file_property(_lang ${_first} LANGUAGE)
     if(_lang STREQUAL Fortran)
-        if (NOT CUDA_LINK_WITH_NVCC)
+        #Check to see if HIP is enabled before resetting 
```

Comment:
Suggestion/optional: something like "Don't reset the linker if the NVCC or HIP linker is required" might be more clear here

Reply:
Done!